graviti/dataframe/column/series.py

```diff
@@ -85,6 +87,37 @@ def __getitem__(self, key: Union[int, Iterable[int]]) -> Any:
     def __len__(self) -> int:
         return self._data.__len__()
 
+    def __repr__(self) -> str:
+        indices = list(self._get_repr_indices())
+        indice_width = len(max(map(str, indices)))
+
+        body = []
+        body_item_width = 0
+        for i in indices:
+            item = self.loc[i]
+            name = item._repr_folding() if hasattr(item, "_repr_folding") else str(item)
+            body.append(name)
+            body_item_width = max(len(name), body_item_width)
+
+        lines = []
+        for indice, value in zip(indices, body):
+            lines.append(f"{indice:<{indice_width+2}}{value:<{body_item_width+2}}")
+        if self.__len__() > _MAX_REPR_ROWS:
+            lines.append(f"...({self.__len__()})")
+        lines.append(f"Series('{self.name}')")
```

Comment:
if self.name = None, should we print `Series('None')`? pandas series do not print this